stdlib/public/stubs/Stubs.cpp

```diff
@@ -128,6 +128,26 @@ static locale_t getCLocale() {
 }
 #endif
 
+#ifdef __ANDROID__
+#define uselocale(a) (nullptr)
+static double swift_strtod_l(const char* a, char** b, locale_t c) {
+	return strtod(a, b);
+}
+static float swift_strtof_l(const char* a, char** b, locale_t c) {
+	return strtof(a, b);
```

Comment:
This won't implement the right semantics -- the current locale will affect the output on Android.

Also, please use spaces for indentation.